src/config.js

```diff
@@ -8,5 +8,6 @@ export default {
   // How much of the buffer must be filled before we consider upswitching
   BUFFER_LOW_WATER_LINE: 0,
   MAX_BUFFER_LOW_WATER_LINE: 30,
-  BUFFER_LOW_WATER_LINE_RATE: 1
+  BUFFER_LOW_WATER_LINE_RATE: 1,
+  BACK_BUFFER_LENGTH: 30
```

Comment:
The forward buffer scales with time, so at the beginning it would be 60 (`GOAL_BUFFER_LENGTH + BACK_BUFFER_LENGTH`) and after playing for a bit it would be 90 (`MAX_GOAL_BUFFER_LENGTH + BACK_BUFFER_LENGTH`).

The constants could probably use some renaming to make them more clear :)

Reply:
hmm. Ok, curious if we should consider tweaking these values, but that's just curiosity... 🙃

Reply:
Indeed! I just used what was already in the code.